### PR TITLE
Flying simple animal mobs are immune to chasms

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
rscadd: Flying simple animals, like butterflies, parrots and guardians are now immune to falling into chasms.
/:cl:

### CODE REVIEW

**Comment by xxalpha** [comment]
Why don't all mobs have a flying attribute?

**Comment by coiax** [comment]
Don't ask me, I didn't write it.

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -104,6 +104,9 @@ var/global/list/parasites = list() //all currently existing/living guardians
 /mob/living/simple_animal/hostile/guardian/Life() //Dies if the summoner dies
 	..()
 	update_health_hud() //we need to update our health display to match our summoner and we can't practically give the summoner a hook to do it
+
+	float(1) // No matter our situation, guardians always float up and down
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: pls no

Comment (coiax) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: what's the better way?

**Comment by drynwyn** [comment]
What about space carp? They can very clearly fly.

**Comment by Supermichael777** [comment]
no space carp are swimming in the interstellar medium and chasms act as a whirlpool

File: code/modules/mob/living/simple_animal/guardian/guardian.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: I mean this will look like butts don't do it

Comment (coiax) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: well I suppose you're in charge of holos.

**Pull request merged**